### PR TITLE
Acknowledgement chapter for The Turing Way handbook

### LINKED ISSUE #1034: Governance: Create policy for authorship and contribution credits
### Summary

* How can we fairly recognise contributions of people?

### What needs to be done?

- [x] Compile the scenario where this idea is applicable and what rewards can we think of
- [ ] Invite different solutions from the community
- [ ] Create a policy that aligns with the values of team and community members

### Who can help?

* @KirstieJane 
* Community members

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Partially Fixes #1034 
Next step is to create the community architecture

### List of changes proposed in this PR (pull-request)

* Creating a chapter for acknowledging contributions
  * [x] Landing file 1: community-handbook/acknowledgement
  * [x] Subchapter on how we acknowledge members: community-handbook/acknowledgement/acknowledgement-members
  * [x] Subchapter on how we record their acknowledgement: community-handbook/acknowledgement/acknowledgement-records

#### Update

- The acknowledgment chapter was written with the practice of acknowledging, persona and pathway for different acknowledgments
- In a discussion with @KirstieJane
- The file was split into multiple files across three chapters
  - Earlier all the three chapters were in this PR but now 2 of them are moved to separate PRs #1252  and #1254 for easing the review load
  - This PR still holds the chapters and subchapters on acknowledgment as listed above

### What should a reviewer concentrate their feedback on?

- [x] If all the aspects are covered
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
## Update Log

- The acknowledgment file was written based on the discussion with @KirstieJane on the practice of acknowledging 
- The first draft of persona and pathway for different acknowledgments was created by Kirstie
- A longer version was written by Malvika
-  After discussion with Kirstie, that file was split into multiple files across three chapters
  - Earlier all the three chapters were in this PR but now 2 of them are moved to separate PRs #1252 (persona creation) and #1254 (personas and pathways in The Turing Way) for distributing the review work
- This PR still holds the chapters and subchapters on acknowledgment as listed below
  - Landing file 1: community-handbook/acknowledgement
  - Subchapter on how we acknowledge members: community-handbook/acknowledgement/acknowledgement-members
  - Subchapter on how we record their acknowledgement: community-handbook/acknowledgement/acknowledgement-records

(This report is updated in the issue details)

**Comment by rainsworth** [comment]
Hi @malvikasharan, I am taking a look at this today!

**Comment by malvikasharan** [comment]
@rainsworth Thank you so much. I have moved the persona contents here: #1254 (personas and pathways in The Turing Way) - the draft that you created for persona and pathway has been added there as a subchapter, and there is a small chapter in #1252 (a guide to persona creation).

**Comment by malvikasharan** [comment]
Thanks again for the review Rachael. It's really useful to have your opinion on these chapters. 🙌

File: book/website/community-handbook/acknowledgement/acknowledgement-members.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Community Members and Shared Ownership
+
+## Community members, contributors and co-authors
+
+As a community-led project, we believe in and promote a collaborative approach to develop the content of this book.
+Writing a chapter is an iterative process that involves designing, outlining, writing, reviewing, and editing its contents.
+Similarly, community-oriented processes evolve over time with the help and feedback from the community.
+Everyone who contributes to this book, no matter how small or big their contributions are, is recognised in this project as a contributor and a community member.
+
+All the substantial contributors are named as co-authors on the book as a whole.
```
Comment (mloning) [inline book/website/community-handbook/acknowledgement/acknowledgement-members.md:10]: What's the process for determining what a substantial contribution is, who is involved in that? Can that process itself be changed?

File: book/website/community-handbook/acknowledgement/acknowledgement-examples.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+- *Persona for helping gain funding*: Ismael shared a funding opportunity with an intention to financially support the development of the guide on ethical research.
+They also helped in editing a grant proposal.
+  - **Acknowledgement:** Ismael is acknowledged with 🔍 (`fundingFinding`) emoji in the Contributors table and 💵 (`financial`) emoji for their support and work.
+
+### Project Management
+
+Team members, core contributors and community members with sustained engagements often take on project management tasks and support _The Turing Way_ community and project as a whole.
+  - **Acknowledgement:** These members are acknowledged with 📆 (`ProjectManagement`) emoji in the Contributors table.
+
+*Did we miss listing a contribution type? Please create an issue on the GitHub repository to discuss that with the team members.*
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/community-handbook/acknowledgement/acknowledgement-examples.md:270]: Definitely useful. Thanks Markus. 🎊

File: book/website/community-handbook/acknowledgement/acknowledgement-members.md (hunk 1 above)
Comment (malvikasharan) [reply book/website/community-handbook/acknowledgement/acknowledgement-members.md:10]: In the example subchapter, we explain what kind of contributions are counted towards authorship. However, I agree that this word doesn't help and we should explain what we mean by "substantial".

Comment (mloning) [reply book/website/community-handbook/acknowledgement/acknowledgement-members.md:10]: I just saw that you actually make it more precise in the `GOVERNANCE.md`:
> "Substantial" is a subjective term. As the project is quite small at the moment, the lead investigator Dr Kirstie Whitaker, will decide whether a contribution or collection of contributions is substantial enough to be given authorship. She can be contacted by email at kwhitaker@turing.ac.uk.

Perhaps it make sense to link that here! Of course there is still the question on how and under what conditions that decision process could be changed and whether some other decision process may be more appropriate. I don't have definite answers to these questions and can't find much guidance on them either, but they seem to be important, no?

Comment (mloning) [reply book/website/community-handbook/acknowledgement/acknowledgement-members.md:10]: I mean not just the decision whether something is "substantial" or not but more generally how decision are made.

Comment (malvikasharan) [reply book/website/community-handbook/acknowledgement/acknowledgement-members.md:10]: Good point. I am still working on this chapter and can add a section on community process and workflow - something that is currently missing.

Comment (malvikasharan) [reply book/website/community-handbook/acknowledgement/acknowledgement-members.md:10]: Reading more carefully now. I think I will draft the process and workflow chapter separately. There are a few more community aspects that I need to document where we will have more on governance. Though, that should definitely be cross referenced here.

Comment (mloning) [reply book/website/community-handbook/acknowledgement/acknowledgement-members.md:10]: We're in the process of drafting a governance document in sktime, perhaps that's helpful, feedback is of course also appreciated :) https://github.com/alan-turing-institute/sktime/pull/324

**Comment by malvikasharan** [comment]
Thanks to Sarah, Rachael, Patricia, and Louise, we have updated the contributors.md file. I have also created a combined copy of the contributor's record in the afterword section with a small UNIX script, which we discuss if that should be included in the workflow. I also faced some issues with the bad Latin ci check, as it doesn't really consider the ignore list of files to exempt. I will open an issue and leave that for Sarah when she returns. 

I will go ahead and merge this to the book, and happy for further review online.

**Pull request merged**